packages/example-app-base/src/components/Page/Page.module.scss

```diff
@@ -78,8 +79,10 @@ $PageSubHeadMargin: 20px;
     @include ms-fontSize-16;
   }
 
-  p {
-    margin-bottom: 12px;
+  p,
+  pre {
+    margin-top: 0;
+    margin-bottom: 1.2em;
```

Comment:
Need to get this spacing approved

Reply:
![image](https://user-images.githubusercontent.com/5864305/57166341-b9c5dd00-6dae-11e9-9eaf-20bab891dd14.png)